stock-managment/src/main/java/org/dhis2/android/rtsm/ui/managestock/ManageStockViewModel.kt

```diff
@@ -76,23 +80,31 @@ class ManageStockViewModel @Inject constructor(
     val operationState: LiveData<OperationState<LiveData<PagedList<StockItem>>>>
         get() = _networkState
 
+    private val _allTableState = MutableStateFlow<List<TableModel>>(mutableListOf())
+    private val allTableState: StateFlow<List<TableModel>> = _allTableState
+
     private val _screenState: MutableLiveData<TableScreenState> = MutableLiveData(
         TableScreenState(emptyList(), false)
     )
     val screenState: LiveData<TableScreenState> = _screenState
 
+    private val _stockItems: MutableLiveData<PagedList<StockItem>> =
+        MutableLiveData<PagedList<StockItem>>()
+
+    private val errors = MutableStateFlow<MutableMap<String, String>>(mutableMapOf())
```

Comment:
I think you don't need this: I think error handling can be solved in one part. I'll try to explain with comments. Read below please: